### ISSUE TITLE
Migrations trace backwards registers

### ISSUE THREAD

**Issue by maurobpt** [phalcon/migrations#66]
Hi i was testing migrations from different branches onto development and this happened

```
'1582539287636860_0000100','2020-03-10 11:52:33','2020-03-10 11:52:33'
'1582539471102635_0000100','2020-03-10 11:52:33','2020-03-10 11:52:33'
'1582560343774480_0000100','2020-03-10 11:52:33','2020-03-10 11:52:34'
'1582712574839699_0000100','2020-03-10 11:52:34','2020-03-10 11:52:34'
'1582737171386285_0000100','2020-03-10 11:52:34','2020-03-10 11:52:34'
'1582738412346286_0000100','2020-03-10 11:52:34','2020-03-10 11:52:34'
'1583253607256544_0000100','2020-03-10 11:52:34','2020-03-10 11:52:34'
'1583774030720287_0000100','2020-03-10 11:52:34','2020-03-10 11:52:34'
'1583921033737407_0000100','2020-03-11 10:05:01','2020-03-11 10:05:01'
```

But there was one missing
`1583773658223923_0000100`

Executing on console 
`vendor/bin/phalcon-migrations migration run --config=migrations.php`
the result was
```
Phalcon Migrations                               
  Info: Everything is up to date  
```
The migration that wasn't running had a previous date than my migration
`1583774030720287_0000100`

**Comment by Jeckerson** [comment]
Fixed in #69

**Comment by Jeckerson** [comment]
Released in `v1.2.1`
